code/modules/mapping/mapping_helpers.dm

```diff
@@ -205,3 +205,22 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 		CRASH("Wrong disease type passed in.")
 	var/datum/disease/D = new disease_type()
 	return list(component_type,D)
+
+/obj/effect/mapping_helpers/dead_body_placer
+	name = "Dead Body placer"
+	icon_state = "deadbodyplacer"
+
+/obj/effect/mapping_helpers/dead_body_placer/Initialize(mapload)
+	var/area/a = get_area(src)
+	var/trays = list()
+	for (var/i in a.contents)
+		if (istype(i, /obj/structure/bodycontainer/morgue))
+			trays += i
+	for (var/i = 1 to 2)
```

Comment:
How about a `#define CORPSES_ON_ROUNDSTART` ?

Reply:
make it a var on the mapping_helper so you can manually edit it if your morgue is bigger/smaller than "normal"

Reply:
^ better idea

Reply:
Cobby's suggestion is done on the upcoming change.